notice_and_comment/templates/regulations/comment-disclaimer.html

```diff
@@ -45,7 +46,7 @@
           will waive any CBI claims for the information submitted. To view any additional
           information for submitting comments, such as anonymous or sensitive submissions,
           refer to the specific Federal Register notice on which you are commenting and the
-          website of the department or agency. More specifically, please see the ADDRESSES
+          website of the department or agency. More specifically, please see the <a href="{% url 'chrome_preamble' meta.document_number %}" target="_blank">ADDRESSES</a>
```

Comment:
I went with new window since it was deep in the comment review process and worried about someone finding their way back there but I could go either way.